core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -974,6 +974,22 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     assert(manager.isZombie)
   }
 
+  test("check uniqueness of TaskSetManager name") {
+    sc = new SparkContext("local", "test")
+    sched = new FakeTaskScheduler(sc, ("exec1", "host1"))
+    val taskSet = FakeTask.createTaskSet(1, 0, 0)
```

Comment:
Can you pass these in as named parameters, to make the tests a little easier to read?

taskSet = FakeTask.createTaskSet(numTasks = 1, stageId = 0, stageAttemptId = 0)